base_uom_merge/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Base Product UOMs Merge",
+    "summary": "Merge duplicate product UOMs",
+    "version": "15.0.1.0.0",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "category": "Warehouse Management",
```

Comment:
I would change the category field, as this module is not exclusively for Warehouse Management. It can be used from the Sales application without requiring the use of a warehouse, as it no depends to "stock"

Maybe change to category "Product".

Reply:
Update category as per dependent module Product's category.

Thanks for suggestion